lms/djangoapps/student_account/views.py

```diff
@@ -378,6 +379,9 @@ def account_settings_context(request):
                 'options': released_languages(),
             }, 'level_of_education': {
                 'options': [(choice[0], _(choice[1])) for choice in UserProfile.LEVEL_OF_EDUCATION_CHOICES],  # pylint: disable=translation-of-non-string
+            }, 'dropdown': {
+                # pylint: disable=translation-of-non-string
```

Comment:
You should do this disable inline (as L381) so you only disable the single line w/ the issue